resources/js/components/items/FeedbackForm.vue

```diff
@@ -150,18 +178,13 @@ export default {
                 'hidden': !this.hidden,
             }
 
-            let feedback = await window.axios.post('/api/feedbacks', data);
-
-            this.$emit('created', feedback.data);
+            window.axios.post('/api/feedbacks', data)
+                .then(this.handleSuccess)
+                .catch(this.handleError);
```

Comment:
Aqui, poderia ser feito utilizando `async`/`await`, da seguinte forma:
```suggestion
try {
    let response = await window.axios.post('/api/feedbacks', data);
    this.handleSuccess(response);
} catch(error) {
    this.handleError(error);
}
```

Mantendo a função `createFeedback()` como assíncrona. Desta forma acredito que o problema com o _delay_ seja solucionado também. Talvez o efeito visual não apareça por estarmos utilizando o servidor localmente, aí acaba ficando bem rapidinho. Para visualizar o efeito (fazendo desta forma assíncrona) talvez seja necessário adicionar um _delay_ no servidor, antes dele enviar a _response_.

Reply:
Interesting, realmente não tinha pensado nisso, hei de implementar. Assim que puder eu implemento e dou push. Muito obrigado pela sugestão 😃